ChangeLog.md

```diff
@@ -1,1095 +1,1124 @@
-   Azure Files.
-1. Fixed issue where application crashed after being throttled by Azure Storage Service. (The
-   primary fix here is for Page Blobs, but a secondary part of the fix also helps with Block Blobs.)
-1. Fixed functionality and usabilty issues with `remove` command
-1. Improved performance for short-duration jobs (e.g. those lasting less than a minute)
-1. Prevent unnecessary error message that sometimes appeared when cancelling a job
-1. Various improvements to the online help and error messages.
-
-
-## Version 10.0.8:
-
-1. Rewrote sync command to eliminate numerous bugs and improve usability (see wiki for details)
-1. Implemented various improvements to memory management
-1. Added MD5 validation support (available options: NoCheck, LogOnly, FailIfDifferent, FailIfDifferentOrMissing)
-1. Added last modified time checks for source to guarantee transfer integrity 
-1. Formalized outputs in JSON and elevated the output flag to the root level
-1. Eliminated outputs to STDERR (for new version notifications), which were causing problems for certain CI systems
-1. Improved log format for Windows
-1. Optimized plan file sizes
-1. Improved command line parameter names as follows (to be consistent with naming pattern of other parameters):
-   1. fromTo -> from-to
-   1. blobType -> blob-type
-   1. excludedBlobType -> excluded-blob-type
-   1. outputRaw (in "list" command) -> output
-   1. stdIn-enable (reserved for internal use) -> stdin-enable
+
+# Change Log
+
+## Version 10.27.0
+
+### New Features
+1. Added ability to specify concurrency of piped uploads via the AZCOPY_CONCURRENCY_VALUE environment variable. ([#2821](https://github.com/Azure/azure-storage-azcopy/pull/2821))
+2. Update message returned when transfer failure is related to tags permission issues. ([#2798](https://github.com/Azure/azure-storage-azcopy/issues/2798)) 
+3. Optimized number of calls to create directory. ([#2828](https://github.com/Azure/azure-storage-azcopy/pull/2828))
+
+### Dependency updates
+1. Golang 1.22.5 -> 1.23.1
+2. azblob 1.4.0 -> 1.4.1
+3. azfile 1.3.0 -> 1.3.1
+4. azdatalake 1.2.0 -> 1.2.1
+5. azidentity 1.7.0 -> 1.8.0
+6. azcore 1.13.0 -> 1.16.0
+
+### Bug Fixes
+1. Fixed an issue where piped downloads in Linux would append AzCopy version information if on an older version. ([#2774](https://github.com/Azure/azure-storage-azcopy/pull/2774))
+2. Fixed an issue where sync with --hash-meta-dir would cause original files to be hidden. ([#2758](https://github.com/Azure/azure-storage-azcopy/issues/2758)) 
+3. Fixed an issue where jobs list would result in a segmentation violation. ([#2714](https://github.com/Azure/azure-storage-azcopy/issues/2714))
+4. Fixed an issue where list for File would not allow OAuth. ([#2821](https://github.com/Azure/azure-storage-azcopy/pull/2821))
+5. Fixed an issue where folders would be handled improperly on datalake accounts. ([#2809](https://github.com/Azure/azure-storage-azcopy/pull/2809))

```

Comment:
This is inaccurate; this is on FNS. HNS works fine.

Reply:
oops for some reason my brain read HNS instead of FNS. Will fix.